models/employees/employee.py

```diff
@@ -1,12 +1,13 @@
 from dataclasses import dataclass
+import uuid
 
 from .employee_priority_enum import EmployeePriorityEnum
 from .employee_status_enum import EmployeeStatusEnum
 
 
 @dataclass
 class Employee:
-    name: str                                   # the name of the employee
-    employee_priority: EmployeePriorityEnum     # the priority of the employee
-    status: EmployeeStatusEnum                  # the status of the employee
-    id: int                                     # the employee id
+    name: str
+    priority: EmployeePriorityEnum
+    status: EmployeeStatusEnum
+    id: uuid.UUID
```

Comment:
id is scary, use `employee_id` because its a keyword. both `id` and `type` should be prefixed